tests/h/groups/util_test.py

```diff
@@ -25,6 +25,10 @@ def test_pubid(self, group):
 
     def test_is_public(self, group):
         assert group.is_public
+        assert group.pubid == '__world__'
```

Comment:
This seems to be a remnant from the merge conflict. This assertion already exists in the previous test method, so this can be safely deleted.